templates/image.jsx

```diff
@@ -34,7 +34,10 @@ export default function Image(props) {
     >
 
       <img
-        className={prefixClasses(props.classNamePrefixes, ['__image'])}
+        className={classes([
+          prefixClasses(props.classNamePrefixes, ['__image']),
+          props?._srcFocalPoint && `object-position-${props?._srcFocalPoint}`
```

Comment:
Hi Guy, this seems like a very specific class name to apply, but I can't see anywhere in Core or Vanilla that currently picks it up. Are you intending to use it to apply the [CSS object-position](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) setting somewhere automatically?

Reply:
Hey Cahir, you're correct it's not currently being used by any plugins. The intention is to bake in the potential for changing the images object position going forwards should we require it.

Reply:
I think the ability to set the `object-position` is a useful additional. I think implementing this as a `class` though could be limiting.
1, the classes need to be predefined,
2, typically you cover keyword values as classes (`top`, `left`, `bottom`, `right`) ruling out percentage, length and off set values. Even with those keywords I tend to double up e.g. `right bottom`.

For Vanilla background image `background-position`, we set [CSS styles](https://github.com/adaptlearning/adapt-contrib-vanilla/blob/59e377d7a2da38ab29c596a278b7ce8374dfbe52/js/themeView.js#L70) rather than classes. What are thoughts on doing the same here?

Reply:
My two cents... We could create classes for all of the keyword values. For instance:

`object-position-top-left`
`object-position-top-right`
`object-position-bottom-left`
`object-position-bottom-right`
etc....

Then, custom theme work can be utilized for more granular control over the positioning (ex. `left 4% bottom 2%`). However, I think the keyword options would fit the majority of use cases.